### PR TITLE
Document preparing interfaces for "slow network" testing

### PR DESCRIPTION
**Pull request by istankovic**
The first few commits are misc fixes for things I ran into while building the docs.
Note that I didn't include the new file in any toctree as I am not sure where to put it. Suggestions more than welcome.

### CODE REVIEW

File: docs/slow-network-test.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
+Testing Raiden on a slow network
+================================
+
+In order to analyze Raiden performance on slow networks, it is useful to have a way of
```
Comment (ulope) [inline docs/slow-network-test.rst:4]: Can you please add a sentence that this is linux specific and also mention which tools / packages the example script requires?

File: docs/slow-network-test.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+introducing latencies. On Linux this can be achieved by making use of the tc utility.
+
+The following script can be used to prepare the interfaces:
+
+.. code:: bash
+
+    #!/usr/bin/env bash
+
+    IFACE="wlan0"
```
Comment (ulope) [inline docs/slow-network-test.rst:13]: Maybe add a comment that this needs to be changed to whichever interface is the default one in the user's machine.

File: docs/slow-network-test.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+interfaces (veth-b) into a newly created network namespace. After setting up the interfaces'
+addresses and adding the default route, the script will configure packet forwarding to the
+real interface (wlan0) connected to the Internet.
+
+Once the interfaces are prepared, we can simulate a network delay of, say, 200ms by
+using tc:
+
+.. code:: bash
+
+    tc qdisc add dev ${VETHB} root netem delay 200ms
```
Comment (ulope) [inline docs/slow-network-test.rst:53]: Nitpick: `${VETHB}` isn't defined here.

**Review by ulope — APPROVED** [review]
LGTM

That's a nice addition.
I think it should fit ok in the `Other` section.

It's very true however that our docs could use a general restructuring.

File: docs/slow-network-test.rst (hunk 1 above)
Comment (istankovic) [reply docs/slow-network-test.rst:4]: Done.

File: docs/slow-network-test.rst (hunk 2 above)
Comment (istankovic) [reply docs/slow-network-test.rst:13]: Done.

File: docs/slow-network-test.rst (hunk 3 above)
Comment (istankovic) [reply docs/slow-network-test.rst:53]: The command is intended to be run apart from the script, but is in fact tied to the script in the sense that whatever interface is used for `VETHB`, the same interface should be used in this `tc` invocation and I wanted to make that clear. I've added a few words to further clarify this.

**Pull request merged**